test/assume_role/agent_configs/config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "agent": {
```

Comment:
This is not the role_arn for sending metrics to another account?

Reply:
Are you tagging to a right line? or are you saying there should be another config in agent's config instead having creds file path in `/opt/aws/amazon-cloudwatch-agent/etc/common-config.toml`?

Reply:
There is a way to assume role directly for sending metrics or logs to a different account. If this is a lift and shift from the internal test, I don't really care which way we do it.